sr_speech_control/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_speech_control</name>
+  <version>0.0.0</version>
+  <description>The sr_speech_control package</description>
```

Comment:
Should it be something more informative like:
> The package for controlling robot with voice commands
rather than just repeating package name?